src/VideoReceiver/GStreamer.cc

```diff
@@ -133,18 +133,22 @@ blacklist()
     changeRank("avdec_h264", GST_RANK_MARGINAL + 1);
```

Comment:
That's not correct and breaks normal user experience.

Reply:
'Default' option should not change any ranks.

Reply:
In reference also to @andrewvoznytsa comment [here](https://github.com/mavlink/qgroundcontrol/pull/9213/files#discussion_r533461326)

I think we should
- remove this line, do nothing to lower the rank of any decoder
- if user selects 'Default' - we do nothing to change the ranks
- if user selects 'Force *' - we set the rank of the forced decoder PRIMARY + 1.

Reply:
> 'Default' option should not change any ranks.

Default, in the actual patch. Does not change any rank, the early return will be used.

Reply:
Done.